src/test/test-time-util.c

```diff
@@ -701,6 +701,17 @@ static void test_parse_timestamp_impl(const char *tz) {
         test_parse_timestamp_one("70-01-01 00:00:01.001 UTC", 0, USEC_PER_SEC + 1000);
         test_parse_timestamp_one("70-01-01 00:00:01.0010 UTC", 0, USEC_PER_SEC + 1000);
 
+        /* Examples from RFC3339 */
+        test_parse_timestamp_one("1985-04-12T23:20:50.52Z", 0, 482196050 * USEC_PER_SEC + 520000);
+        test_parse_timestamp_one("1996-12-19T16:39:57-08:00", 0, 851042397 * USEC_PER_SEC + 000000);
+        test_parse_timestamp_one("1996-12-20T00:39:57Z", 0, 851042397 * USEC_PER_SEC + 000000);
+        test_parse_timestamp_one("1990-12-31T23:59:60Z", 0, 662688000 * USEC_PER_SEC + 000000);
+        test_parse_timestamp_one("1990-12-31T15:59:60-08:00", 0, 662688000 * USEC_PER_SEC + 000000);
+        // test_parse_timestamp_one("1937-01-01T12:00:27.87+00:20", 0,  -1041337173 * USEC_PER_SEC + 870000);
```

Comment:
leftover?

Reply:
If refusing timestamps from before the epoch is expected – https://github.com/systemd/systemd/pull/29049#issuecomment-1706974982 – then yes, but I wanted to confirm it is before I removed it.

Reply:
I think so, yes

Reply:
Removed with a note